instrumentation/all/opentelemetry-instrumentation-all.gemspec

```diff
@@ -77,4 +77,6 @@ Gem::Specification.new do |spec|
     spec.metadata['bug_tracker_uri'] = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues'
     spec.metadata['documentation_uri'] = "https://rubydoc.info/gems/#{spec.name}/#{spec.version}"
   end
+
+  spec.post_install_message = File.read(File.expand_path('../../POST_INSTALL_MESSAGE', __dir__))
```

Comment:
I see the post-install message is on `all` and `rails`, but not the other instrumentation gems. Do you think most users have these gems installed, so we don't need to add the message to every instrumentation library?

Reply:
That tradeoff makes sense to me!